rules/CodeQuality/Rector/If_/SimplifyIfNullableReturnRector.php

```diff
@@ -223,11 +223,11 @@ private function removeAndReturn(Return_ $return, Expression $expression, Expr $
         $this->removeNode($return);
         $this->removeNode($expression);
 
-        $return = new Return_($expr);
+        $exprReturn = new Return_($expr);
```

Comment:
Return_ is stmt, so should be `stmtReturn`

Reply:
The `expr` is information about the inner node:

E.g. 

```php
$stringReturn = new Return(new String(...));
```